HW_4/Task_3.js

```diff
@@ -0,0 +1,14 @@
+const minAge = 18;
+const maxAge = 60;
+let age = parseInt("11");
+if(typeof age !== "number") {
+    console.log("Incorrect data type");  
+} else if(age < minAge) {
+    console.log("You don't have access cause your age is " + age + "\nIt's less than " + minAge);    
+} else if (age >= minAge && age < maxAge) {
+    console.log("Welcome!");
+} else if (age > maxAge) {
+    console.log("Keep calm and look Culture channel");    
+} else {
+    console.log("Technical work");    
+}
```

Comment:
В данном задании необходимо использовать `isNaN` в `if` вьіражении
Если `true` - приводим значение к `number` дабьі не вьіполнять преобразования в дальнейших if ветвлениях
И внимательно смотрим наши ветвления

```suggestion
let age = "11";
if(isNaN(age)) {
    console.log("Incorrect data type");  
} else {
       if(age < minAge) {
       console.log("You don't have access cause your age is " + age + "\nIt's less than " + minAge);    
    } else if (age >= minAge && age < maxAge) {
         console.log("Welcome!");
    } else if (age > maxAge) {
       console.log("Keep calm and look Culture channel");    
    } else {
       console.log("Technical work"); 
    }   
}
```